### PR TITLE
add functionality to delete question

### PR DESCRIPTION
**Pull request by Tayef-Shah**
There's currently an issue where the Material UI button isn't firing the onclick function every time.

### CODE REVIEW

File: src/Components/FormCreation/FormSection.js [hunk 1]
```diff
@@ -161,13 +180,15 @@ function FormSection({
           />
         ) : null}
       </CardWrapper>
-      {sectionData.questions.map((_question, questionKey) => (
+      {questions.map((_question, questionKey) => (
         <CardWrapper key={questionKey}>
           <FormCard
             card={sectionData.questions[questionKey]}
             key={questionKey + "_question"}
+            card={questions[questionKey]}
```
Comment (ahmedhamodi) [inline src/Components/FormCreation/FormSection.js:188]: Duplicate card property

**Review by ahmedhamodi — COMMENTED** [review]
Can you rebase on master? Some of these changes have already been made. Also, the travis CI is failing because of a duplicate card property which I pointed out.

File: src/Components/FormCreation/FormSection.js (hunk 1 above)
Comment (Tayef-Shah) [reply src/Components/FormCreation/FormSection.js:188]: Oh true

**Review by ahmedhamodi — APPROVED** [review]
LGTM if travis CI passes!

**Review by GodGreg — APPROVED** [review]

**Pull request merged**